.github/workflows/tf_apply_staging.yml

```diff
@@ -0,0 +1,94 @@
+name: "Terraform apply staging"
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "terragrunt/**"
+      - ".github/workflows/tf_apply_staging.yml"
+
+env:
+  TERRAFORM_VERSION: 1.0.3
+  TERRAGRUNT_VERSION: 0.38.4
+  TF_VAR_api_auth_token: ${{ secrets.STAGING_API_AUTH_TOKEN }}
+  TF_VAR_aws_org_id: ${{ secrets.STAGING_AWS_ORG_ID }}
+  TF_VAR_rds_password: ${{ secrets.STAGING_RDS_PASSWORD }}
+  TF_VAR_slack_webhook_url: ${{ secrets.SCAN_FILES_STAGING_OPS_WEBHOOK }}
+  AWS_REGION: ca-central-1
+
+permissions:
+  id-token: write
+  contents: read
+  pull-requests: write
+  actions: write
+  checks: write
+  statuses: write
+
+jobs:
+  terragrunt-apply:
+    if: github.ref == 'refs/heads/main' && github.event_name == 'push'
```

Comment:
I don't think we need this anymore since we only trigger this workflow on push to main.

Reply:
removed!